Makefile

```
@@ -78,3 +79,11 @@ lint:
 .PHONY: tidy
 tidy:
 	$(MAKE) --no-print-directory -C misc     tidy
+
+$(BUILDDIR)/:
+	mkdir -p $(BUILDDIR)/
+
+.PHONY: govulncheck
+govulncheck: $(BUILDDIR)/
+	GOBIN=$(BUILDDIR) go install golang.org/x/vuln/cmd/govulncheck@latest
```

niptick: How about check if it's already installed and only install it if it's not?
```suggestion
	@if [ ! -f $(BUILDDIR)/govulncheck ]; then \
		GOBIN=$(BUILDDIR) go install golang.org/x/vuln/cmd/govulncheck@latest
	fi
```